livekit-agents/livekit/agents/tts/tts.py

```diff
@@ -69,8 +69,12 @@ def __init__(self, *, streaming_supported: bool) -> None:
         self._streaming_supported = streaming_supported
 
     @abstractmethod
-    async def synthesize(self, *, text: str) -> AsyncIterable[SynthesizedAudio]:
-        pass
+    async def synthesize(self, text: str, **_: Any) -> AsyncIterable[SynthesizedAudio]:
```

Comment:
We shouldn't add , **_: Any

Reply:
I see the TTS of OpenAI has more arguments. The base class should not allow for more arguments.
So the change needs to be made in the OpenAI TTS, not the base class

Reply:
Hrmm so is the preferred pattern to make new TTS objects for new configurations (i.e. put configuration in the constructor)?

Reply:
actually as I think about it more, I think you're right and requiring configuration in the ctor makes more sense even if it requires more TTS objects per-configuration. Will make the adjustments in a PR soon